<ui version="4.0" >
 <class>PasswordFormClass</class>
 <widget class="QDialog" name="PasswordFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PasswordForm</string>
  </property>
  <widget class="QPushButton" name="pbOK" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>70</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Please input password</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
